<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaPrincipal</class>
 <widget class="QMainWindow" name="TelaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RsPapers</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="TopButtons">
      <layout class="QHBoxLayout" name="Header">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QToolButton" name="abrir">
         <property name="text">
          <string>Abrir</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../styles/light/abrir.png</normaloff>../styles/light/abrir.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="salvar">
         <property name="text">
          <string> Salvar</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../styles/light/salvar.png</normaloff>../styles/light/salvar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="exportar">
         <property name="text">
          <string>Exportar</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../styles/light/exportar.png</normaloff>../styles/light/exportar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="novo_projeto">
         <property name="text">
          <string>Novo Projeto</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../styles/light/novo.png</normaloff>../styles/light/novo.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="preferencias">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Preferências</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../styles/light/preferencias.png</normaloff>../styles/light/preferencias.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QCheckBox" name="modo_escuro">
         <property name="text">
          <string>Modo Escuro</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Section">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="MenuLateral">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QToolButton" name="ordenar">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Ordenar</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../styles/light/ordernar.png</normaloff>../styles/light/ordernar.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>40</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="artigos_selecionados">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string> Seleção de Artigos 
 Total: 2.000</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../styles/light/buscar.png</normaloff>../styles/light/buscar.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="artigos_interesse">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string> Artigos de Interesse 
 Total: 10</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../styles/light/aprovar.png</normaloff>../styles/light/aprovar.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="artigos_descartados">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string> Artigos Descartados 
 Total: 5</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../styles/light/descartar.png</normaloff>../styles/light/descartar.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QToolButton" name="estatisticas">
          <property name="text">
           <string>Estatísticas</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../styles/light/estatisticas.png</normaloff>../styles/light/estatisticas.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_artigos_por_pagina">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Artigos por Página:</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLineEdit" name="artigos_por_pagina">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>10</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_ordenados">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Artigos Ordenados:&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;2.000&lt;/p&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="article">
        <item alignment="Qt::AlignHCenter">
         <widget class="QWidget" name="paginacao" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="_2">
           <item>
            <widget class="QToolButton" name="pag_primeira">
             <property name="icon">
              <iconset>
               <normaloff>../styles/light/left_twice.png</normaloff>../styles/light/left_twice.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="pag_anterior">
             <property name="icon">
              <iconset>
               <normaloff>../styles/light/left.png</normaloff>../styles/light/left.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="pagina">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Página</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="pag_atual">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>1</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="total">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>de 250</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="pag_prox">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../styles/light/right.png</normaloff>../styles/light/right.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="pag_ultima">
             <property name="icon">
              <iconset>
               <normaloff>../styles/light/right_twice.png</normaloff>../styles/light/right_twice.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="Artigos">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="RolagemArtigos">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>445</width>
             <height>350</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="bottomMargin">
             <number>20</number>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
